manifests/4.3/ptp-operator.v4.3.0.clusterserviceversion.yaml

```diff
@@ -41,6 +41,15 @@ metadata:
             ]
           }
         },
+        {
+          "apiVersion": "ptp.openshift.io/v1",
+          "kind": "NodePtpDevice",
+          "metadata": {
+            "name": "worker.node.com",
```

Comment:
how about to use 'example.com' for the name, to align with https://tools.ietf.org/html/rfc6761?

Reply:
Done